src/app/containers/MostRead/Canonical/LastUpdated.test.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { latin } from '@bbc/gel-foundations/scripts';
+import LastUpdated from './LastUpdated';
+
+let container;
+
+describe('MostReadCanonical - LastUpdated', () => {
+  beforeEach(() => {
+    container = document.createElement('div');
+    document.body.appendChild(container);
+  });
+
+  afterEach(() => {
+    container = null;
+  });
+
+  it(`test LastUpdated renders when timestamp > 60 days`, () => {
+    ReactDOM.render(
+      <LastUpdated
+        timestamp={864691200}
+        prefix="Last Updated: "
+        script={latin}
+        service="news"
+        locale="en-gb"
+      />,
+      container,
+    );
+
+    expect(container.querySelector('time').textContent).toEqual(
+      'Last Updated:  11 January 1970',
+    );
+  });
+
+  it(`test LastUpdated does not renders when timestamp < 60 days`, () => {
```

Comment:
```suggestion
  it(`LastUpdated does not render when timestamp < 60 days`, () => {
```

Reply:
Reworded to make it more consistent with other tests that we have:
https://github.com/bbc/simorgh/blob/afa5969a3f3202a4115b05b9433037e2d25bbfbc/src/app/containers/MostRead/Canonical/LastUpdated.test.jsx#L18